<!-- TODO: Insert components of component -->
<launch>
    <node pkg="planning" type="dev_global_route.py" name="DevGlobalRoute" output="screen">
           <param name="sampling_resolution" value="75.0" />
           <param name="routes" value="/opt/leaderboard/data/routes_devtest.xml" />
           <param name="role_name" value="hero" />
    </node>
    <node pkg="planning" type="preplanner.py" name="PrePlanner" output="screen">
           <param name="role_name" value="hero" />
    </node>
</launch>
